helpdesk_mgmt/controllers/main.py

```diff
@@ -20,14 +22,22 @@ def support_ticket_close(self, **kw):
                 values[field_name] = int(field_value)
             else:
                 values[field_name] = field_value
-        ticket = (
-            http.request.env["helpdesk.ticket"]
-            .sudo()
-            .search([("id", "=", values["ticket_id"])])
-        )
-        ticket.stage_id = values.get("stage_id")
-
-        return werkzeug.utils.redirect("/my/ticket/" + str(ticket.id))
+        if "ticket_id" in values:
```

Comment:
Why is this needed and is it related to the change of this PR?

Reply:
What do you mean, adding the check for `ticket_id` ?

Reply:
Yes, why it was not needed and now it is? And if it's an unrelated patch, it shouldn't go here.

Reply:
Anyway, I think the 15.0 change should be backported instead.